### DISCUSSION TITLE
personal access tokens

### DISCUSSION THREAD

**Discussion by rhubarbdog** [community/community#37231] [New to GitHub] [answered]
i've just regenerated my personal access token and it didn't display or i missed it.

what do i do now?

thanks

**Comment by godnondsilva** [comment]
I believe you can regenerate it once again right? You could also create a new personal access token.

**Comment by msuliq** [comment]
You can generate a new token or I believe you should be able to see the previous token that you missed in your GitHub account settings. Please see instructions here https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

**Comment by DenverCoder1** [comment, marked as answer]
The new token should appear above the form in a blue box after you click the "Regenerate Token" button.

![image](https://user-images.githubusercontent.com/20955511/197865830-86f5ba1c-ec66-43ba-acdb-83f92660049a.png)

**Reply by Payne-Cong** [reply]
There is no information about the token, only permissions and notes, nothing is displayed, I have tried many times

**Comment by rhubarbdog** [comment]
Thank done it
100% good advice

**Reply by DSDmark** [reply]
Hey @rhubarbdog,
I hope you got your answer. If you got your answer, then you should close that question by click accept answer button, or you can write the answer and accept own answer.

**Comment by TANGCONGKinh** [comment]
echo "# error_subcode" >> README.md 
git init 
git add README.md 
git commit -m "first commit" 
git branch -M main 
git remote add origin https://github.com/TANGCONGKinh/error_subcode.git
 git push - nguồn gốc chính của u

**Comment by Mehver** [comment]
Same issue, I tried lots of times, but everytime after I click "Generate Token" then the page jump back to the token list. No token value shows up...

I used token before, and I believe this is a functional bug. Might caused by the Bata function?

**Reply by Mehver** [reply]
Update: Reload cookie solved this problem. Btw, it is the second time I reload GitHub's cookie to solve bugs today, last time was to solve an error log when I want to generate a new release.

I'm using the latest version of Chrome and I wonder is this my personal issue? Or lots of others also need to reload their cookies to solve problems often? Developers should look at it.
